pkg/api/controllers/gitprovider/gitprovider.go

```diff
@@ -40,6 +40,7 @@ func ListGitProviders(ctx *gin.Context) {
 
 	for _, provider := range response {
 		provider.Token = ""
+		*provider.SigningKey = ""
```

Comment:
Better to do `provider.SigningKey = nil` otherwise a nil-pointer exception can occur.

Reply:
Done ✅